from datetime import datetime

import mysql.connector

def alquilar_auto(cliente, auto, fecha_alquiler, fecha_devolucion, servicio, local):
    try:
        connection=mysql.connector.connect(host='localhost',database='rent_cars',user='root',password='admin')
        mySql_insert_query = '''INSERT INTO productos (Cliente, Autos, Order_date, Fecha_alquiler, Fecha_devolucion, Servicio, Locale) 
                                VALUES (%s, %s, %s, %s, %s, %s, %s) '''

        cursor = connection.cursor()
        current_Date = datetime.now()
        formatted_date = current_Date.strftime('%d-%m-%Y %H:%M:%S')
        insert_tuple = (cliente, auto, formatted_date, fecha_alquiler, fecha_devolucion, servicio, local)

        cursor.execute(mySql_insert_query, insert_tuple)
        connection.commit()
        print("Reserva creada exitosamente")

    except mysql.connector.Error as error:
        connection.rollback()
        print("Error al crear la reserva {}".format(error))

    finally:
        if connection.is_connected():
            cursor.close()
            connection.close()
            print("MySQL connection is closed")
